oas_apivideo.yaml

```diff
@@ -13335,7 +13365,7 @@ paths:
           - `device-type`: Returns analytics based on the type of device used by the viewers. Response values can include: `computer`, `phone`, `tablet`, `tv`, `console`, `wearable`, `unknown`.
           - `operating-system`: Returns analytics based on the operating system used by the viewers. Response values can include `windows`, `mac osx`, `android`, `ios`, `linux`.
           - `browser`: Returns analytics based on the browser used by the viewers. Response values can include `chrome`, `firefox`, `edge`, `opera`.
-          - `referrer`: TODO
+          - `referrer`: Returns the URL where the view originates from, for example a website where the video is embedded. View events from Android and iOS return empty strings as referrer.
```

Comment:
Last sentence should maybe be: "View events from Android and iOS return empty strings as the [or a] referrer."